projects/templates/admin/change_list.html

```diff
@@ -0,0 +1,17 @@
+{% extends "admin/change_list.html" %}
+
+{% load i18n %}
+{% load admin_urls %}
+
+{# Original template renders object-tools only when has_add_permission is True. #}
+{# This hack allows sub templates to add to object-tools #}
+{% block object-tools %}
+  <ul class="object-tools">
+    {% block object-tools-items %}
+      <li><a class='export_link' href="{% url 'admin:calender_export' %}{{cl.get_query_string}}">{% trans "Export dates" %}</a></li>
+      {% if has_add_permission %}
+        {{ block.super }}
+      {% endif %}
+    {% endblock %}
+  </ul>
+{% endblock %}
```

Comment:
This leads to the behaviour that the link is present on all admin pages and not only on the `admin/projects/project/` subpath. I would expect to find this link only there.

Reply:
[a0d551b](https://github.com/gocept/ok_tools/pull/88/commits/a0d551bb168cba04ab1a8b6e84cbb77875f8fe47)